spec/html-proofer/links_spec.rb

```diff
@@ -3,6 +3,10 @@
 require "spec_helper"
 
 describe "Links test" do
+  unicode_file = File.join(FIXTURES_DIR, "links", "erstiebegrüßung.html")
+  before(:all) { File.open(unicode_file, "w") {} }
+  after(:all) { File.delete(unicode_file) }
```

Comment:
`before(:all)` only executes once before all specs in the suite is executed. I think this is convenient because like `ensure`, `after(:all)` is executed after all specs regardless of how they fare.